webrtc/stun/stun_utils.nim

```diff
@@ -0,0 +1,57 @@
+# Nim-WebRTC
+# Copyright (c) 2024 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+import sequtils, typetraits, std/sha1
+import bearssl
+
+proc generateRandomSeq*(rng: ref HmacDrbgContext, size: int): seq[byte] =
```

Comment:
it is supposed to return a `seq`, but I believe it doesn't return anything, potentially `nil`.

Reply:
What do you mean? I initialize `result`. It definitely returns a seq.

Reply:
I find it confusing. There are 3 different ways of returning in Nim and it's used arbitrarily.